src/main/java/org/broadinstitute/consent/http/resources/DaaResource.java

```diff
@@ -222,4 +222,60 @@ public Response sendDaaRequestMessage(
       return createExceptionResponse(e);
     }
   }
+
+  @POST
+  @Consumes(MediaType.APPLICATION_JSON)
+  @RolesAllowed({ADMIN, CHAIRPERSON})
+  @Path("/bulk/{daaId}")
+  public Response bulkAddUsersToDaa(
+      @Auth AuthUser authUser,
+      @PathParam("daaId") Integer daaId,
+      String json) {
+    try {
+      User authedUser = userService.findUserByEmail(authUser.getEmail());
+      List<User> users = userService.findUsersInJsonArray(json, "users");
+      if (authedUser.hasUserRole(UserRoles.SIGNINGOFFICIAL)) {
+        for (User user : users) {
+          if (!Objects.equals(authedUser.getInstitutionId(), user.getInstitutionId())) {
+            return Response.status(Status.FORBIDDEN).build();
+          }
+        }
+      }
+      daaService.findById(daaId);
+      for (User user : users) {
+        libraryCardService.addDaaToUserLibraryCardByInstitution(user, authedUser.getInstitutionId(), daaId);
+      }
```

Comment:
There is a potential future optimization here. This looped service call ends up making looped database calls. It would be more efficient to bulk add in a single database operation but we can tackle that when we need to.